archlinuxcn/llvm-git/PKGBUILD

```diff
@@ -172,6 +172,7 @@ package_compiler-rt-git() {
     pkgdesc="Compiler runtime libraries for clang (git version)"
     url="https://compiler-rt.llvm.org/"
     depends=("llvm-git=$pkgver-$pkgrel" 'gcc-libs')
+    depends_x86_64=('lib32-gcc-libs')
```

Comment:
Is this really a dependency? Or is it just a optional one. I assume you don't need the lib32 version if you are just using the x64 libraries.

Reply:
It seemingly asks for it in namcap.